### DISCUSSION TITLE
希望支持特定app可以使用特定的配置。

### DISCUSSION THREAD

**Discussion by tiancer** [MetaCubeX/ClashMetaForAndroid#233] [Ideas]
比如我有多份clash配置。
其中
- 配置A，高速稳定，但流量较贵。
- 配置B，不稳定，打开个网页或连接个什么东西，可能得转个几圈，但连接成功后，偶尔速度也挺快的，用来下载和看视频都足够，且流量便宜。
- 我的clash是始终开启的，希望平常使用场景用配置A，特定app使用场景用配置B。
- 又或者其他人可能比如说有些节点支持网飞，gpt，有些不支持，可能也需要这样的功能。

**Comment by mengdegege** [comment]
手搓配置文件，把所有的订阅都写进一个配置再分流就可以了
